chaincode/.openzeppelin/unknown-31337.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
Is this folder supposed to be ignored?

Reply:
Good question.
Normally it would **not** be ignored as it tracks the addresses of the proxies deployed on each network.
However it is a little tough to read, and we can manually control the deployments.
So we've also added logging into the deploy scripts to tell us the deployed addresses.